app/storage/encrypted_storage.py

```diff
@@ -30,7 +28,7 @@ def store(self, data, user_id, user_ik):
     def get(self, user_id, user_ik):
```

Comment:
Pycharm hinted to a potential issue here, user_ik is define as a optional argument in storage medium but it's not here. That makes it firstly confusing and secondly it can potentially break if someone calls it assuming optional.

Given that it makes sense to make it mandatory given the only place it's used defines the argument